term-utils/script.c

```diff
@@ -408,6 +409,9 @@ static int log_start(struct script_control *ctl,
 			fprintf(log->fp, _("<not executed on terminal>"));
 
 		fputs("]\n", log->fp);
+
+		fprintf(log->fp, _("%s"), ctl->command);
```

Comment:
`ctl->command` can be NULL, can't it?

Reply:
I doubt `ctl->command` is translatable.

Reply:
`ctl->command` just carries the command from the `-c command` option. Hence it is directly dependt on the content in the command option.

It is initialized with NULL. However, I'm not aware of use cases not using the -c option. If there exists such use cases, it should take that fact into account.